pkg/tekton/syntax/pipeline.go

```diff
@@ -1884,7 +1884,7 @@ func (p *PipelineOverride) AsStepsSlice() []*Step {
 
 // MatchesPipeline returns true if the pipeline name is specified in the override or no pipeline is specified at all in the override
 func (p *PipelineOverride) MatchesPipeline(name string) bool {
-	if p.Pipeline == "" || p.Pipeline == name {
+	if p.Pipeline == "" || strings.EqualFold(p.Pipeline, name) {
```

Comment:
Never seen this one before ;-) Had to look up what this method does. 

So we generally don't want to care about capitalisation? So 'RELEASE', 'Release' or 'release' is all the same for us?

Reply:
I'd prefer to care about case, but given that we have `pullRequest` in the syntax but `pullrequest` as the actual kind...we're kind of stuck with case insensitivity.

Reply:
> I'd prefer to care about case,

+1

> but given that we have pullRequest in the syntax but pullrequest as the actual kind...we're kind of stuck with case insensitivity.

Wonder whether there would be a way to sort this out. I can already seeing this biting us back one day.